### ISSUE TITLE
Find token using Flamite

### ISSUE THREAD

**Issue by HugoSart** [defaultnamehere/tinder-detective#48]
Is it possible to find my auth token using Flamite for Chrome? I'm having issues trying to find the token, so I thought maybe that way could be easier.

I'm using Web Sniffer extension to watch the traffic, but I didn't find the right token.

**Comment by ghost** [comment]
An easier way is to just use Firefox/Explorer (Chrome is buggy for this)
Log in to the target FB account and remove the Tinder App
Open a new tab and inspect.. monitor the network traffic
Goto [this link](https://www.facebook.com/v2.6/dialog/oauth?redirect_uri=fb464891386855067%3A%2F%2Fauthorize%2F&state=%7B%22challenge%22%3A%22q1WMwhvSfbWHvd8xz5PT6lk6eoA%253D%22%2C%220_auth_logger_id%22%3A%2254783C22-558A-4E54-A1EE-BB9E357CC11F%22%2C%22com.facebook.sdk_client_state%22%3Atrue%2C%223_method%22%3A%22sfvc_auth%22%7D&scope=user_birthday%2Cuser_photos%2Cuser_education_history%2Cemail%2Cuser_relationship_details%2Cuser_friends%2Cuser_work_history%2Cuser_likes&response_type=token%2Csigned_request&default_audience=friends&return_scopes=true&auth_type=rerequest&client_id=464891386855067&ret=login&sdk=ios&logger_id=54783C22-558A-4E54-A1EE-BB9E357CC11F#_=_)
Click through the okay's until you get to a blank white screen
Goto inspector and search for "ACCESS_TOKEN"
There's your token

For the FacebookID, just look for the c_user param in the cookie for the oauth call.

**Comment by hmc-crane-s17** [comment]
@smelsworst does that link create a Tinder account linked to the facebook account?

**Comment by ghost** [comment]
@hmc-crane-s17 yep (from memory).. though it's been a while since I did this. that said, there were a number of encoding, etc errors that I faced and I was never bothered enough to resolve them to get the app to work. so, don't expect this to be the last hurdle you face when using this app.
